system-tests/tests/smoke/cre/cmd/environment/environment.go

```diff
@@ -105,7 +106,10 @@ var startCmd = &cobra.Command{
 			if p != nil {
 				fmt.Println("Panicked when starting environment")
 				if err, ok := p.(error); ok {
-					fmt.Fprint(os.Stderr, errors.Wrap(err, "error:\n%s").Error())
+					fmt.Fprint(os.Stderr, "Error:")
+					fmt.Fprint(os.Stderr, err)
+					fmt.Fprint(os.Stderr, "Stack trace:")
+					fmt.Fprint(os.Stderr, string(debug.Stack()))
```

Comment:
You should be able to chain these (only a space is inserted between), and maybe add a leading new line before the stacktrace?
```suggestion
					fmt.Fprint(os.Stderr, "Error:", err)
					fmt.Fprint(os.Stderr, "\nStack trace:", string(debug.Stack()))
```